styles.css

```diff
@@ -0,0 +1,585 @@
+  color: black;
+}
+
+.info {
+  padding-top: 1vw;
+  background: url(./capstoneimgs/diamond.png) repeat;
+  background-color: rgb(26, 26, 26);
+}
+
+.main-program {
+  padding-top: 3vw;
+  padding-bottom: 3vw;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.program-cards-title {
+  color: #ec5242;
+  text-decoration: none;
+}
+
+.pc-text {
+  color: white;
+}
+
+.title {
+  color: white;
+  font-size: 15px;
+  padding-bottom: 0.5vw;
+  background: url(./capstoneimgs/title_underline.png) bottom no-repeat;
+}
+
+h3 {
+  font-size: 18px;
+  color: #ec5242;
+}
+
+.program-cards {
+  display: grid;
+  grid-template-columns: 100px 100px auto;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  padding: 0.5em;
+  background-color: rgba(65, 66, 70, 0.4);
+}
+
+.join-inti {
+  padding: 1.5em;
+  margin: 4em;
+  width: 75%;
+  color: white;
+  background-color: #ec5242;
+}
+
+.desktop-see-program {
+  display: none;
+}
+
+.main-performers {
+  height: auto;
+}
+
+#perTitle {
+  text-align: center;
+  color: black;
+  font-size: 15px;
+  margin-bottom: 0;
+}
+
+.red-line {
+  width: 100%;
+  height: auto;
+  background: url('../capstoneimgs/title_underline.png') no-repeat;
+}
+
+.container-performers {
+  /* padding: 1rem 6rem 6rem 6rem; */
+}
+
+#all-performers {
+  flex-wrap: wrap;
+  margin-bottom: 0.5rem;
+
+  /* padding-inline: 3em; */
+  height: 630px;
+  max-width: 1200px;
+  overflow: hidden;
+}
+
+/* img{
+  width: 100%;
+  height: auto;
+} */
+
+.card {
+  width: 85%;
+  flex-direction: row;
+}
+
+.card img {
+  width: 180px;
+  height: 180px;
+
+  /* object-fit: center; */
+  object-position: center;
+  padding-top: 30px;
+}
+
+@media (min-width: 768px) {
+  .card {
+    width: 550px;
+  }
+}
+
+.performer-card {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2em;
+}
+
+.checkerbg {
+  background: url('../capstoneimgs/checkboardimg.jpg') top left no-repeat;
+  background-size: contain;
+  justify-self: center;
+  padding: 0.7em 0 0 0.7em;
+  width: 3.5em;
+  height: 3.5em;
+}
+
+.band-detail {
+  display: flex;
+  flex-direction: column;
+
+  /* margin-top: 9vw;
+  margin-left: 12vw; */
+}
+
+.d-flex {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.band-brief-info {
+  font-size: 16px;
+  padding-bottom: 10px;
+  color: #ec5242;
+  background: url(./capstoneimgs/title_underlinegray.svg) bottom left no-repeat;
+}
+
+.artist-intro {
+  font-size: 16px;
+  padding-top: 15px;
+  padding-bottom: 10px;
+}
+
+.more-less {
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  margin: 4em;
+  width: 75%;
+  color: black;
+  height: auto;
+  background-color: white;
+  border: 1px solid #d3d3d3;
+  padding: 0.8em;
+}
+
+.more-less::after {
+  display: flex;
+  transition: 0.2s;
+  font-size: 22px;
+  text-align: center;
+  align-self: center;
+  align-items: center;
+}
+
+.more-lessExpand {
+  height: auto !important;
+  overflow: visible !important;
+}
```

Comment:
- Hey, as suggested by the previous reviewer [here](https://github.com/Maedea9/capstone-m1/pull/3/files/1332d8d364cd7085207356e5f74b9a48814a086a#r1142819953), let's a better way of applying styles instead of using the `!important` keyword. please, fix this. they are many that's why he only highlighted one. You should fix this on all occurece in styles except if you're using a CSS framework.

Reply:
Yes, @zieeco thank you, I am aware of it. I explained in the loom video too that I have used this for the MORE-LESS button to work, it's the way I found it would work. If not I will remove it as this button was optional.